tests/AUTOMATE_TEST_GUIDE.md

```diff
@@ -0,0 +1,17 @@
+# Create new autoamted test case - Guide
+
+A structure that can be used to have an idea about the creation of a new automated test case.
+
+## Base files to use
```

Comment:
I like the concept of the base files to use, but I also think it could be beneficial if you explain why we prefer to separate that functionality into separate classes (and folders). Do you think it could help other engineers the separation of concerns you're suggesting to do here? So they can see any new functionality related with forms, should go into a separate file with X naming convention, in Y folder. Same for contacts, etc, and so at the end the *.wdio.spec.js file looks like this: (then you provide a code example in this guide, not just the link)

Reply:
Thanks! I added a note that I think explains a little better the purpose, let me know if you think that I should add something else.